### PR TITLE
App Switcher Button Added

### PR DESCRIPTION
**Pull request by joachim-junior**

### CODE REVIEW

File: src/components/navbar/navbar.component.tsx [hunk 1]
```diff
@@ -55,6 +57,12 @@ const Navbar: React.FC<NavbarProps> = ({
     }
   };
 
+  const [switcherToggler, setSwitcherToggler] = React.useState(false);
+
+  const handleSwitcherClick = React.useCallback(() => {
+    setSwitcherToggler(switcherToggler => !switcherToggler);
+  }, [switcherToggler]);
```
Comment (FlorianRappl) [inline src/components/navbar/navbar.component.tsx:64]: This dependency is not needed. You only capture `setSwitcherToggler` in the callback, and this one is by definition stable.

File: src/components/navbar/navbar.component.tsx [hunk 2]
```diff
@@ -94,6 +102,35 @@ const Navbar: React.FC<NavbarProps> = ({
...
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
+                  aria-label="app switcher"
+                  aria-labelledby="app swithcer Icon"
+                  name="switcher"
+                >
+                  <AppSwitcher20
+                    onClick={handleSwitcherClick}
+                    style={{ display: switcherToggler ? "none" : "block" }}
```
Comment (FlorianRappl) [inline src/components/navbar/navbar.component.tsx:112]: I am not sure why you "block" and "none".

Just have the DOM adjusted.

```
{React.createElement(switchToggler ? AppSwitcher20 : Close20, ...)}
```

If you prefer JSX then just introduce a variable and use it

```
const Icon = switchToggler ? AppSwitcher20 : Close20
```

**Comment by brandones** [comment]
Hey @joachim-junior , would you mind adding a screenshot for this work, or linking it to a ticket if there is one? Just so it's clear what "app switcher button" refers to in the UI.

**Comment by joachim-junior** [comment]
> Hey @joachim-junior , would you mind adding a screenshot for this work, or linking it to a ticket if there is one? Just so it's clear what "app switcher button" refers to in the UI.

Yeah sure, I can do that. I'll add that in the comments. I was actually implementing the changes requested by @FlorianRappl. Sorry for the late reply.

**Comment by brandones** [comment]
No worries at all, and anything less than 24 hours is absolutely not a "late reply" :)

**Comment by FlorianRappl** [comment]
I guess replaced by #60. Please reuse existing PRs and don't open new PRs just for changes / addressing review comments. Thanks!

**Pull request closed without merging**